### DISCUSSION TITLE
Managing Relationships in the Form Builder

### DISCUSSION THREAD

**Discussion by nettsite** [filamentphp/filament#10619] [Help] [answered]
I have put this on Discord, I don't want to be a nuisance and repeat it all here, so I am just sharing the link to the post:

https://discord.com/channels/883083792112300104/1191758594929070101

I really need some help, I would be grateful for anything.

**Comment by leandrocfe** [comment]
Are you able to share a minimal repo on the Github to reproduce this issue?

**Reply by nettsite** [reply, marked as answer]
Hi Leandro, thanks for reaching out. I have partially solved this with some help from Dan Harrin.

The trick is to make the foreign key column in the child record nullable.
